pageserver/src/page_service.rs

```diff
@@ -140,19 +191,25 @@ pub async fn libpq_listener_main(
         }
     }
 
-    debug!("page_service loop terminated");
+    debug!("page_service listener loop terminated");
 
-    Ok(())
+    Connections {
+        cancel: connections_cancel,
+        tasks: connection_handler_tasks,
+    }
 }
 
+type ConnectionHandlerResult = anyhow::Result<()>;
```

Comment:
I do not understand why.

Reply:
Me neither, probably I had an intermediate refactoring where I attempted to move away from anyhow::Result